src/app/dashboard/dashboard.component.html

```diff
@@ -76,7 +72,7 @@ <h4>Project</h4>
                         <div class="status">
                           <div class="success ng-hide" ng-show="projectsbuild.lastBuild.worker.status == 'Succeeded'"><i class="icon ion-checkmark-circled status-success"></i></div>
                           <div class="failed" ng-show="projectsbuild.lastBuild.worker.status == 'Failed'"><i class="icon ion-close-circled status-fail"></i></div>
-                          <div class="active ng-hide" ng-show="projectsbuild.lastBuild.worker.status == 'Active'"><img src="assets/images/rolling.svg" class="status-active"></div>
+                          <div class="active ng-hide" ng-show="projectsbuild.lastBuild.worker.status == 'Active'"><img src="../assets/images/rolling.svg" class="status-active"></div>
```

Comment:
Awesome to see the active spinners! Anything under the assets directory i is available from an absolute path when fetching from Angular, so this should be`/assets/images/rolling.svg`.